analyzer/core/src/main/scala/io/scalac/newspaper/analyzer/core/Analyzer.scala

```diff
@@ -0,0 +1,53 @@
+package io.scalac.newspaper.analyzer.core
+
+import scala.collection.mutable
+
+class Analyzer {
+
+  import Analyzer._
+
+  private val archive: mutable.Map[String, PreprocessedContent] = new mutable.HashMap()
+
+  private val commentPattern = """(?s)<!--.*?-->""".r
+  private val tagPattern = """(?s)<(/?)\s*([a-zA-Z0-1-]+)[^>]*>""".r
+  private val scriptPattern = """(?s)<script>(.*?)</script>""".r
+  private val bodyPattern = """(?s)<body>(.*)</body>""".r
+
```

Comment:
Haven't seen them yet in our sample data. Good call, but I remove all attributes already, so any iframe would be reduced to ```<iframe>```.

...or do you mean we should fetch the `iframe` in case the articles are embedded that way? I hope no-one does that, it would be awful (and a bit problematic for us).